api/python/quilt3/data_transfer.py

```diff
@@ -62,7 +61,7 @@ def _copy_local_file(ctx, size, src_path, dest_path):
     ctx.progress(size)
     shutil.copymode(src_path, dest_path)
 
-    ctx.done(pathlib.Path(dest_path).as_uri())
+    ctx.done(PhysicalKey(None, dest_path, None))
```

Comment:
It might be clearer to use either a constructor method like `fromPath()` or named arguments + default. Having two positional arguments as None is not very intuitive.

Reply:
Yep. I only added `from_path` later, and missed some of the constructor calls.

I wish Python had a way to make a private constructor (without jumping through a bunch of hoops.)